test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -1574,20 +1916,21 @@ def tearDown(self):
         conf = {'enabled': 'False', 'freq': 30, 'event': events}
         self.server.manager(MGR_CMD_SET, HOOK, conf, self.hook_name)
         # Cleanup any temp file created
-        self.logger.info("Deleting temporary files %s" % self.tempfile)
-        self.du.rm(hostname=self.serverA, path=self.tempfile,
-                   force=True)
+        self.logger.info('Deleting temporary files %s' % self.tempfile)
+        self.du.rm(hostname=self.serverA, path=self.tempfile, force=True)
         # Cleanup frozen jobs
         if 'freezer' in self.paths:
-            self.logger.info("Cleaning up frozen jobs ****")
+            self.logger.info('Cleaning up frozen jobs ****')
```

Comment:
Is this a debug message that forgot to be removed?

Reply:
Status versus debug... in this case I'm leaning toward the former, but it's a judgement call either way. I'd prefer to leave it as-is.